packages/desktop-gui/src/specs/specs-list.jsx

```diff
@@ -156,6 +165,28 @@ class SpecsList extends Component {
     return spec.hasChildren ? this._folderContent(spec, nestingLevel) : this._specContent(spec, nestingLevel)
   }
 
+  _setFocus = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _setBlur = () => {
+    this.setState({
+      isFocused: !this.state.isFocused,
+    })
+  }
+
+  _isMac () {
+    return (window.clientInformation['platform'] === 'MacIntel')
+      ? 'Press Cmd + F to make a quick search...'
+      : 'Press Ctrl + F to make a quick search...'
+  }
+
+  _togglePlaceholderSearchTips = () => {
+    return (this.state.isFocused) ? 'Search' : this._isMac()
+  }
```

Comment:
- Rename function/variables to describe what it's returning. 
- Reduce redundant text strings.

```suggestion
  _searchPlaceholderText () {
 	osKey = window.clientInformation['platform'] === 'MacIntel' ? 'Cmd' : 'Ctrl'
 	
    return `Press ${osKey} + F to make a quick search...`
  }

  _togglePlaceholderSearchTips = () => {
    return (this.state.isFocused) ? 'Search' : this._searchPlaceholderText()
  }
```

Reply:
@jennifer-shehane I have addressed the comments & pushed the latest changes.